src/commands/math/advancedSymbols.js

```diff
@@ -322,5 +323,11 @@ LatexCmds.cap = LatexCmds.intersect = LatexCmds.intersection =
 // FIXME: the correct LaTeX would be ^\circ but we can't parse that
 LatexCmds.deg = LatexCmds.degree = bind(VanillaSymbol,'\\degree ','&deg;');
 
+LatexCmds.celsius = bind(VanillaSymbol,'\\celsius ','&#8451;');
+LatexCmds.micro = bind(VanillaSymbol,'\\micro ','&#181;');
+LatexCmds.perthousand = bind(VanillaSymbol,'\\perthousand ','&#8240;');
+LatexCmds.ohm = bind(VanillaSymbol,'\\ohm ','&#8486;');
```

Comment:
Oh... I see... You thought degree was not standard... Also, not sure what the rest of the comment was about because ^\circ is rendered correctly (although different than the \degree macro, since it's literally a circle displayed as an exponent... Anyway, I'm going to clean this up and put only the changes for the unary operator in this pull request and I'll submit another PR for these other changes.